Library/Homebrew/test/utils/svn_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "utils/svn"
+
+describe Utils do
+  describe "#self.svn_available?" do
+    before(:each) do
+      if described_class.instance_variable_defined?(:@svn)
+        described_class.send(:remove_instance_variable, :@svn)
+      end
+    end
+
+    it "returns svn version if svn available" do
+      expect(described_class.svn_available?).to be_truthy
+    end
+  end
+
+  describe "#self.svn_remote_exists" do
+    it "returns true when svn is not available" do
+      allow(Utils).to receive(:svn_available?).and_return(false)
+      expect(described_class.svn_remote_exists("blah")).to be_truthy
+    end
+
+    context "when svn is available" do
+      before do
+        allow(Utils).to receive(:svn_available?).and_return(true)
+        url.mkpath
+      end
+
+      let(:url) { mktmpdir/"blah" }
```

Comment:
Why does this need to use `mktmpdir` and not write to an already existing location that's deleted after tests: for example: https://github.com/Homebrew/brew/blob/2c43e069c25e3c7cc255f5cd105df3597246b952/Library/Homebrew/test/spec_helper.rb#L110-L129